views/tags.ejs

```diff
@@ -0,0 +1,7 @@
+<h1>Tags</h1>
+
+<form method="POST">
+	<input type="text" name="tag" placeholder="Enter tag here">
+	<input type="hidden" name="id" value="<%= favorite.id %>">
+	<input type="submit" value="Submit">
+</form>
```

Comment:
After this form, we can list all tags associated with this favorite (make sure you un-comment-out the include in your backend!)
```
<% favorite.tags.forEach(function(tag){ %>
	<p><a href="/favorites/tags/<%= tag.id %>"><%= tag.tag %></a></p>
<% }); %>
```